src/Tests/CoinDetails.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
- Good job on implementing the snapshots test for components. but, 
   - [ ] Kindly add unit tests for pure functions in your Redux code as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20unit%20tests%20for%20pure%20functions%20in%20your%20Redux%20code.).
   - [ ] Please add integration tests for your application using the React Testing Libraryas per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20integration%20tests,the%20Redux%20Store.).